iati/core/default.py

```diff
@@ -89,17 +89,22 @@ def codelists(version=None, use_cache=False):
         Add a function to return a single Codelist by name.
 
     """
-    paths = iati.core.resources.get_all_codelist_paths()
+    if version is None:
```

Comment:
These couple of lines are used in several places - may be worth a helper function to do it in 1.